src/rpc/util.cpp

```diff
@@ -209,65 +210,52 @@ CTxDestination AddAndGetMultisigDestination(const int required, const std::vecto
     return dest;
 }
 
-class DescribeAddressVisitor : public boost::static_visitor<UniValue>
-{
-public:
-    explicit DescribeAddressVisitor() {}
+constexpr auto DescribeAddressVisitor = [](const auto& dest) -> UniValue {
```

Comment:
Could you comment on why you chose to change the approach here but not on `DescribeWalletAddressVisitor`?

Reply:
No reason. Simplfied diff by using the same approach everywhere.